lib/view_model/wallet_list/wallet_list_view_model.dart

```diff
@@ -65,6 +83,28 @@ abstract class WalletListViewModelBase with Store {
         ),
       ),
     );
+
+    //========== Split into shared seed groups and single wallets list
+    _walletManager.updateWalletGroups();
+
+    for (var group in _walletManager.walletGroups) {
+      if (group.wallets.length == 1) {
+        final wallet = group.wallets.first;
+        singleWalletsList.add(
+          WalletListItem(
+            name: wallet.name,
+            type: wallet.type,
+            key: wallet.key,
+            isCurrent:
+                wallet.name == _appStore.wallet?.name && wallet.type == _appStore.wallet?.type,
+            isEnabled: availableWalletTypes.contains(wallet.type),
+            isTestnet: wallet.network?.toLowerCase().contains('testnet') ?? false,
```

Comment:
you can add this to `convertWalletInfoToWalletListItem` and use it here
also add the `isTestnet` param to the convert function in `pre_existing_seeds_view_model`